<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QR Code Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon.ico</normaloff>icons/icon.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>QR Code Generator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_url">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_generate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>QR코드로 변환</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_finishtext">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_directory">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>경로 지정</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_madeby">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>185</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Made by. 10216 이지원</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
